cpg_workflows/large_cohort/generate_gene_table.py

```diff
@@ -0,0 +1,513 @@
+import logging
```

Comment:
Is this all new code, or borrowed from gnomAD/seqr? If borrowed it would be good to add some docstring pointers to the original location of the script for any troubleshooting purposes

Reply:
May apply to other scripts in this PR